lib/std/os.zig

```diff
@@ -3169,3 +3169,13 @@ pub fn dn_expand(
     }
     return error.InvalidDnsPacket;
 }
+
+pub fn yield() void {
+    switch (builtin.os) {
+        .windows => _ = windows.kernel32.SwitchToThread(),
+        .linux => _ = assert(linux.sched_yield() == 0),
+        else => if (builtin.link_libc) {
+            assert(std.c.pthread_yield() == 0);
```

Comment:
`pthread_yield` appears to be a gnu extension. But it looks like `sched_yield` is POSIX. And the linux version does not need to be handled specially, because `system` will be aliased to the c namespace or linux namespace as appropriate. I think you can write this function like so:

```zig
pub fn sched_yield() void { 
    if (builtin.os == .windows) {
        _ = windows.kernel32.SwitchToThread();
        return;
    }
    assert(system.sched_yield() == 0);
}
```

Reply:
Sorry, realized these after committing